orangecanvas/document/quickmenu.py

```diff
@@ -178,6 +288,15 @@ def sizeHint(self):
                 height = height * count
             else:
                 height = 0
+
+            # accommodate width for scroll bar width on mac, should transient scrollbars be disabled
+            # System Preferences -> General -> Show scroll bars
+            if sys.platform == "darwin":
+                scroll = self.view().verticalScrollBar()
+                isTransient = scroll.style().styleHint(QStyle.SH_ScrollBar_Transient, widget=scroll)
+                if not isTransient:
+                    width += scroll.style().pixelMetric(QStyle.PM_ScrollBarExtent, widget=scroll)
```

Comment:
This can (and should) be run regardless of the platform. It was actually wrong before. The scroll bar width should have been always included in the size hint.